docs/source/manual/core.rst

```diff
@@ -26,28 +26,43 @@ Dropwizard consists mostly of glue code to automatically connect and configure t
 Organizing Your Project
 =======================
 
-In general, we recommend you separate your projects into three Maven modules: ``project-api``,
-``project-client``, and ``project-application``.
+If you plan on developing a client library for other developers to access your service, we recommend
+you separate your projects into three Maven modules: ``project-api``, ``project-client``, and
+``project-application``.
 
 ``project-api`` should contain your :ref:`man-core-representations`; ``project-client`` should use
 those classes and an :ref:`HTTP client <man-client>` to implement a full-fledged client for your
 application, and ``project-application`` should provide the actual application implementation, including
 :ref:`man-core-resources`.
 
-Our applications tend to look like this:
+To give a concrete example of this project structure, let's say we wanted to create a Stripe_-like
+API where clients can issue charges and the server would echo the charge back to the client.
+``stripe-api`` project would hold our ``Charge`` object as both the server and client want to work
+with the charge and to promote code reuse, ``Charge`` objects are stored in a shared module.
+``stripe-app`` is the Dropwizard application. ``stripe-client`` abstracts away the raw HTTP
+interactions and deserialization logic.  Instead of using a HTTP client, users of ``stripe-client``
```

Comment:
"...and deserialization logic.  Instead of using a HTTP client..."

Double white space after the dot, not sure if it's intentional.